.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,5 @@
 - [ ] closes #xxxx
 - [ ] tests added / passed
+- [ ] passes `black pandas`
 - [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
```

Comment:
what should we be recommending here? Still keep both flake8 and black? 
The problem with this version of the black command is also that it does the full code base and takes a while

Reply:
make black is pretty fast currently so i would just have this